### PR TITLE
Add RTZ audio in chapter 4

### PR DESCRIPTION
**Pull request by YoyoyoChang**
- Add RTZ audio in chapter 4
- Adjust smoke screen location

### CODE REVIEW

File: game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts [hunk 1]
```diff
@@ -103,17 +103,16 @@ function onStart(complete: () => void) {
...
                 const tricksOfTheTrade = riki.GetAbilityByIndex(2);
                 if (tricksOfTheTrade) {
                     riki.CastAbilityOnPosition(riki.GetAbsOrigin().__add(Vector(-200, 100)), tricksOfTheTrade, 0);
                 }
             }),
-            tg.wait(3),
-
-            tg.immediate(_ => playerHero.SetMoveCapability(UnitMoveCapability.GROUND)),
+            tg.audioDialog(LocalizationKey.Script_4_RTZ_getaway, LocalizationKey.Script_4_RTZ_getaway, ctx => ctx[rikiName]),
+            tg.wait(2),
```
Comment (Shushishtok) [inline game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts:115]: Replace the wait with the audioDialog's last argument for the extra delay.

File: game/scripts/vscripts/Sections/Chapter4/SectionWards.ts [hunk 2]
```diff
@@ -187,7 +188,9 @@ function onStart(complete: () => void) {
                 const runDirection = riki.GetAbsOrigin().__sub(playerHero.GetAbsOrigin()).Normalized();
                 riki.MoveToPosition(riki.GetAbsOrigin().__add(runDirection.__mul(800)));
             }),
-            tg.wait(3),
+
+            tg.audioDialog(LocalizationKey.Script_4_RTZ_cya, LocalizationKey.Script_4_RTZ_cya, ctx => ctx[rikiName]),
+            tg.wait(2),
```
Comment (Shushishtok) [inline game/scripts/vscripts/Sections/Chapter4/SectionWards.ts:193]: Same as above

**Review by Shushishtok — APPROVED** [review]

File: game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts (hunk 1 above)
Comment (YoyoyoChang) [reply game/scripts/vscripts/Sections/Chapter4/SectionOutpost.ts]: replaced.

File: game/scripts/vscripts/Sections/Chapter4/SectionWards.ts (hunk 2 above)
Comment (YoyoyoChang) [reply game/scripts/vscripts/Sections/Chapter4/SectionWards.ts]: replaced.

**Pull request merged**